tests/integration/test_studio_live_experiments.py

```diff
@@ -40,3 +40,18 @@ def test_post_to_studio(tmp_dir, dvc, scm, exp_stage, mocker, monkeypatch, tmp):
         "experiment_rev": exp_rev,
         "metrics": {"metrics.yaml": {"data": {"foo": 1}}},
     }
+
+
+@pytest.mark.skip
+def test_post_to_studio_queue(tmp_dir, dvc, scm, exp_stage, mocker, monkeypatch):
```

Comment:
Is there a way to test this without setting up an actual mock HTTP server?

It looks like mocker.patch` doesn't work for code executed via subprocess inside the queue @pmrowla ?

Reply:
If you want to test that it happens in an actual celery run then no, you would have to setup a mocked HTTP server.

It would probably be better to just test that the correct calls are made from `BaseExecutor.reproduce()` during a workspace run though, since the celery run makes the same calls (but from a child process)